src/cr/cube/crunch_cube.py

```diff
@@ -837,3 +838,35 @@ def index(self, axis, weighted=True):
             margin = margin[:, np.newaxis]
 
         return np.true_divide(props, margin)
+
+    @property
+    def z_scores(self):
```

Comment:
could this be called something like `std_resid` or even `standardized_residuals`?

Reply:
I'm fine with `standardized_residuals`, as @rdrivers pointed out in the original PDF.